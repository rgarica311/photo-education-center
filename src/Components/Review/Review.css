.reviewContainer {
  width: 400px;
  height: 200px;
  border-radius: 5px;
  background-color: lightgrey;
  padding: 10px;
}

.reviewBy {
  margin-right: 5px;
  color: red;
}

.reviewContent {
  display: block;
  margin-bottom: 10px
}

@media screen and (max-width: 600px) {
  .reviewsContainer {
    display: flex;
    flex-direction: column;
    align-items: center;
  }

  .reviewContainer {
    width: 80vw;
    height: 30vh;
    margin-top: 20px;
  }

  .reviewContent {
    font-size: .74rem;
  }
}
