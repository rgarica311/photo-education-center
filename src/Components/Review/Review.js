import React, { Component } from 'react';
import './Review.css';

export default function Review(props) {
  return (
    <div className='reviewContainer'>
      <span className='reviewContent'>
        {props.review.content}
      </span>
      <span  className='reviewBy'>
        {props.review.by}
      </span>
      <span className='reviewFrom'>
          {props.review.from}
      </span>
    </div>
  );
};
