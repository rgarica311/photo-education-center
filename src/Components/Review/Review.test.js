import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Review from './Review';

describe(`Review Component`, () => {
  const review = {
    content: 'test content',
    by: 'test by',
    from: 'test from'
  }

  it('renders Review by default', () => {
    const wrapper = shallow(<Review review={review} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });

});
