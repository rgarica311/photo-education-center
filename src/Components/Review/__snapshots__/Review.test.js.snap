// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Review Component renders Review by default 1`] = `
<div
  className="reviewContainer"
>
  <span
    className="reviewContent"
  >
    test content
  </span>
  <span
    className="reviewBy"
  >
    test by
  </span>
  <span
    className="reviewFrom"
  >
    test from
  </span>
</div>
`;
